library/cwm/src/lib/cwm/multi_status_selector.rb

```diff
@@ -447,7 +448,7 @@ def checkbox_label
         if enabled?
           "<a href=\"#{id}#{LABEL_EVENT_ID}\" style=\"#{text_style}\">#{label}</a>"
         else
-          "<span style\"#{text_style}\">#{label}</a>"
+          "<span style=\"#{text_style}\">#{label}</a>"
```

Comment:
Let's not only add the missing `=` but also close the `span` properly :)
```suggestion
          "<span style=\"#{text_style}\">#{label}</span>"
```